documentation/components/tutorial-flow-grid.asciidoc

```diff
@@ -876,15 +891,18 @@ grid.addSortListener(event -> {
 });
 ----
 
-== Styling
+== Styling the Grid
+
+// I found the original content here difficult to understand and have interpreted as best I can
```

Comment:
```suggestion
```
OK, I'll pay extra attention to it then. In the future it might be best to add this comments as review comments in github, instead of actual line comments to the pull request. We probably don't want to leave these to the final documentation files.